src/client/src/ui/spotTile/components/Tile.tsx

```diff
@@ -15,18 +17,24 @@ interface Props {
 
 interface State {
   notional: string
+  disabled: boolean
 }
 
 class Tile extends React.PureComponent<Props, State> {
   state = {
-    notional: '1000000',
+    notional: DEFAULT_NOTIONAL_VALUE,
```

Comment:
Any reason why we are storing the `notional` state at Tile level as a string instead of numeric? I know it was already the case before this PR, but it's a good opportunity to bring up the discussion.

I think it should be the input itself who deals with the commas, the shorthand notation etc, and expose a clean, valid number to the Tile component (unless I'm missing something!)

Reply:
@spcfran I've asked myself the same question and then I realized that the component was using an input field of `text` type and that the user can enter shortcuts, ie: `1k` for `1,000` and `1m` for `1,000,000` and that we use `numeral(notional).format()` to return a string to render a correctly formatted number ie: `1,234,567.89`.

Reply:
Yep makes sense to treat it as a string for that reason, but dealing with the commas, the `1k`/`1m` notation etc. is the responsibility of the `NotionalInput` component right? That component could handle the state as a string (either as a member on its own state, or directly referring to the `inputRef.value` as it is doing right now and is fine), but then call `this.props.updateNotional` with a parsed number. In fact, the only time `updateNotional` is called is right after doing `numeral(inputValue)`, so we could pass whatever number is (if valid) or `null` if not (`numeral("string").value()` returns `null` apparently) to the Tile component. 

No big deal though as it's working perfectly fine right now, so it's probably better to focus in more important things :)